src/postamble.js

```diff
@@ -43,6 +43,9 @@ if (memoryInitializer) {
     memoryInitializer = Module['locateFile'](memoryInitializer);
   } else if (Module['memoryInitializerPrefixURL']) {
     memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
+  } else if (ENVIRONMENT_IS_NODE) {
+    // Look for .mem file in the same folder as script by default
+    memoryInitializer = __dirname + '/' + memoryInitializer;
```

Comment:
Alternatively to make it more generic, perhaps we could have a `Module['documentDirectory']` which would be initialized to `__dirname` on node.js by default, and the above type of code would always read relative directories with respect to `Module['documentDirectory']` if that exists. That way we'd avoid having to put in node.js specifics at each place of use (assuming we will have more of them in the future, e.g. when node.js adds .wasm support?), and if we need a similar thing for SpiderMonkey or other shells, it would be decoupled as well?